packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -177,7 +176,6 @@ class PackagesGetCommand extends FlutterCommand {
     // Get/upgrade packages in example app as well
     if (rootProject.hasExampleApp && rootProject.example.pubspecFile.existsSync()) {
       final FlutterProject exampleProject = rootProject.example;
-      await _runPubGet(exampleProject.directory.path, exampleProject);
```

Comment:
And this is no longer needed because we're always passing `--example` to dart pub get?

Reply:
Nice, can you make this explicit in the comment on line 176?

Reply:
I tried clarifying, hope it reads OK.